test/powershell/Modules/Microsoft.PowerShell.Core/Exec.Tests.ps1

```diff
@@ -15,10 +15,10 @@ Describe 'Switch-Process tests for Unix' -Tags 'CI' {
         $global:PSDefaultParameterValues = $originalDefaultParameterValues
     }
 
-    It 'Exec alias should map to Switch-Process' {
+    It 'Exec function should map to Switch-Process' {
         $alias = Get-Command exec
```

Comment:
for cleanliness any variables that were `$alias` should be `$function` as not to accidentality confuse future readers of the codebase

Reply:
This is just one example & I would expect that all other usage of `$alias` should be changed without me needing to highlight it as it's only a few additional lines

Reply:
Addressed, thanks!

Reply:
good point, will update